docs/en/Image-Manipulation.md

```diff
@@ -0,0 +1,290 @@
+
+* `OptimalCompression`: Indicates whether the optimal compression is enabled or not. (Default: `false`)
+* `IgnoreUnsupportedFormats`: Indicates whether the unsupported formats are ignored or not. (Default: `false`)
+* `Lossless`: Indicates whether the lossless compression is enabled or not. (Default: `false`)
+
+## [ImageSharp](https://github.com/SixLabors/ImageSharp)
+
+Add the `Volo.Abp.Imaging.ImageSharp` NuGet package to your project and use the `AbpImagingImageSharpModule` module.
+
+```csharp
+[DependsOn(typeof(AbpImagingImageSharpModule))]
+public class MyModule : AbpModule
+{
+    //...
+}
+```
+
+### Configuration
+
+#### ImageSharpCompressOptions
+
+`ImageSharpCompressOptions` is used to configure the ImageSharp image compression system. It has the following properties:
+
+* `DefaultQuality`: The default quality of the JPEG and WebP encoders. (Default: `75`)
+* `JpegEncoder`: The JPEG encoder. (Default: `JpegEncoder` with `Quality` set to `DefaultQuality`)
+* `PngEncoder`: The PNG encoder. (Default: `PngEncoder` with `IgnoreMetadata` set to `true` and `CompressionLevel` set to `PngCompressionLevel.BestCompression`)	
+* `WebPEncoder`: The WebP encoder. (Default: `WebPEncoder` with `Quality` set to `DefaultQuality`)
+
+## ASP.NET Core Integration
+
+Add the `Volo.Abp.Imaging.AspNetCore` NuGet package to your project and use the `AbpImagingAspNetCoreModule` module.
+
+> It needs a provider to work. You can use the [Magick.NET](#magicknet), [ImageSharp](#imagesharp) provider or implement your own provider. Provider does not exist, the input stream will be returned as it is.
+
+```csharp
+[DependsOn(typeof(AbpImagingAspNetCoreModule))]
+public class MyModule : AbpModule
+{
+    //...
+}
+```
+
+### CompressImageAttribute
+
+The `CompressImageAttribute` is used to compress the image before requesting the action. IFormFile, IRemoteStreamContent, Stream and IEnumrable<byte> types are supported. It has the following properties:
+
+* `Parameters`: The parameters that are used to configure the image compression system. (Optional. This applies to all parameters if left blank.)
+
+Usage example:
+
+```csharp
+
+[CompressImage]
+[HttpPost]
+public async Task<IActionResult> Upload(IFormFile file)
+{
+    //...
+}
+```
+
+### ResizeImageAttribute
+
+The `ResizeImageAttribute` is used to resize the image before requesting the action. IFormFile, IRemoteStreamContent, Stream and IEnumrable<byte> types are supported. It has the following properties:
+
+* `Parameters`: The parameters that are used to configure the image resize system. (Optional. This applies to all parameters if left blank.)
+* `Width`: The width of the resized image.
+* `Height`: The height of the resized image.
+* `Mode`: The resize mode. (See the [ImageResizeMode](#imageresizemode) section for more information.)
+
+Usage example:
+
+```csharp
+
+[ResizeImage(Width = 100, Height = 100, Mode = ImageResizeMode.Crop)]
+[HttpPost]
+public async Task<IActionResult> Upload(IFormFile file)
```

Comment:
Document which types are supported in addition to IFromFile
https://github.com/abpframework/abp/pull/16586/files#diff-3ee87982a2c1dc9ae4d134aedb49c059be888271380b562964eea1a062242b1dR31

Reply:
It seems I missed. So, ignore this comment.